engine/src/main/java/io/graphqlcrud/SQLQueryBuilderVisitor.java

```diff
@@ -271,67 +273,130 @@ public void visitArgument(Field field, GraphQLFieldDefinition definition, GraphQ
+                break;
+            case "ne":
+                c = left.ne(((BooleanValue) v).isValue());
+                break;
+        }
         return c;
     }
 
     public Condition visitIntValue(Value v, org.jooq.Field left, String conditionName) {
         Condition c = null;
-        if(conditionName.equals("eq"))
-            c = left.eq(((IntValue)v).getValue());
-        else if(conditionName.equals("ne"))
-            c = left.ne(((IntValue) v).getValue());
-        else if(conditionName.equals("lt"))
-            c = left.lt(((IntValue) v).getValue());
-        else if(conditionName.equals("le"))
-            c = left.le(((IntValue) v).getValue());
-        else if(conditionName.equals("gt"))
-            c = left.gt(((IntValue) v).getValue());
-        else if(conditionName.equals("ge"))
-            c = left.ge(((IntValue) v).getValue());
+        switch (conditionName) {
+            case "eq":
+                c = left.eq(((IntValue) v).getValue());
+                break;
+            case "ne":
+                c = left.ne(((IntValue) v).getValue());
+                break;
+            case "lt":
+                c = left.lt(((IntValue) v).getValue());
+                break;
+            case "le":
+                c = left.le(((IntValue) v).getValue());
+                break;
+            case "gt":
+                c = left.gt(((IntValue) v).getValue());
+                break;
+            case "ge":
+                c = left.ge(((IntValue) v).getValue());
+                break;
+        }
         return c;
     }
 
     public Condition visitFloatValue(Value v, org.jooq.Field left, String conditionName) {
         Condition c = null;
-        if(conditionName.equals("eq"))
-            c = left.eq(((FloatValue)v).getValue());
-        else if(conditionName.equals("ne"))
-            c = left.ne(((FloatValue) v).getValue());
-        else if(conditionName.equals("lt"))
-            c = left.lt(((FloatValue) v).getValue());
-        else if(conditionName.equals("le"))
-            c = left.le(((FloatValue) v).getValue());
-        else if(conditionName.equals("gt"))
-            c = left.gt(((FloatValue) v).getValue());
-        else if(conditionName.equals("ge"))
-            c = left.ge(((FloatValue) v).getValue());
+        switch (conditionName) {
+            case "eq":
+                c = left.eq(((FloatValue) v).getValue());
+                break;
+            case "ne":
+                c = left.ne(((FloatValue) v).getValue());
+                break;
+            case "lt":
+                c = left.lt(((FloatValue) v).getValue());
+                break;
+            case "le":
+                c = left.le(((FloatValue) v).getValue());
+                break;
+            case "gt":
+                c = left.gt(((FloatValue) v).getValue());
+                break;
+            case "ge":
+                c = left.ge(((FloatValue) v).getValue());
+                break;
```

Comment:
put in `default` here and throw a `RuntimeException` saying wrong condition same at all other similar places.